crates/shared/src/arguments.rs

```diff
@@ -147,13 +148,21 @@ pub struct Arguments {
     #[clap(flatten)]
     pub tenderly: tenderly_api::Arguments,
 
+    #[clap(flatten)]
+    pub balances: account_balances::Arguments,
+
     #[clap(flatten)]
     pub logging: LoggingArguments,
 
     /// The Ethereum node URL to connect to.
     #[clap(long, env, default_value = "http://localhost:8545")]
     pub node_url: Url,
 
+    /// An Ethereum node URL that supports `eth_call`s with state overrides to
+    /// be used exclusively for trade simulations.
```

Comment:
nit:
> ...to be used exclusively for trade simulations.

Is no longer correct now that balances can also be queried with simulations.